modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/displayUndoDeleteSuccessToast.ts

```diff
@@ -62,4 +70,18 @@ export default function displayUndoDeleteSuccessToast(
 	};
 
 	openToast(openToastSuccessProps);
+
+	Liferay.componentReady(componentId).then(() => {
+		const toastLink = document.getElementById(componentId);
+
+		if (toastLink) {
+			toastLink.focus();
```

Comment:
Why is the toast being focused? Won't this break accessibility if something else is already focused on the page?